docs/development.md

```diff
@@ -0,0 +1,116 @@
+## Running examples locally
+
+The Hw Event Proxy works with [Cloud Event Proxy](https://github.com/redhat-cne/cloud-event-proxy).
+Run cloud-event-proxy sidecar and consumer example from the cloud-event-proxy repo for testing locally.
+
+### Set environment variables
+```
+export NODE_NAME=mynode
```

Comment:
Node name you can read from spec like this as part of deployment.yaml ?
 env:
            - name: NODE_NAME
              valueFrom:
                fieldRef:
                  fieldPath: spec.nodeName

Reply:
example: https://github.com/redhat-cne/cloud-event-proxy/blob/main/examples/consumer/deployment/deployment.yaml#L35

Reply:
This section is for testing locally without cluster deployment.